bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +204,27 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        if contains_url:
+            urls = re.findall(r'(https?://\S+)', body)
+            has_good_url = False
+            for url in urls:
+                pieces = urlparse(url)
+                netloc = pieces.netloc
+                scheme = pieces.scheme
+                path = pieces.path
+                try:
+                    assert all([scheme, netloc, path])
+                    if netloc.find(vcs_used) != -1 and path.find('issues'):
```

Comment:
So, how about just asking the user to specify the host he/she uses instead of VCS?